### DISCUSSION TITLE
Can we get a detailed guide on running this?

### DISCUSSION THREAD

**Discussion by Devinawesley** [emanuelpm/Blade-Mistress#8] [General]
I desperately miss playing Blade Mistress back in the day. I'm so happy I stumbled across this repo, but I'm so unfamiliar with C++ that I'm having trouble getting it to run. I would lick the bottom of a shoe for a guide on how to get this up and running locally.

**Comment by Dismanbuster** [comment]
me too hehe ,i would love to get a dl-able file to simply play bm

**Comment by krunph** [comment]
Hey!! Just found out this repo too. I want to play this again. I played with The_Punisher era.
How to run this game again? thanks
